.gitignore

```diff
@@ -165,3 +165,6 @@ cython_debug/
 
 # VSCode
 .vscode/
+
+# OS X
+.DS_Store
```

Comment:
Minor change just to keep the codebase consistent through our `pre-commit` hooks: if you can add an extra return here, then we should be good to go!

Reply:
Done!